src/formatting.ts

```diff
@@ -1,12 +1,10 @@
-export function formatMoney(value: number, currencyCode = "€", separator: string = ",") {
-    let decimals = Math.ceil(value % 100);
-    // Math.ceil(1_99.99 % 100 = 99.99) = 100
-    // Then we want to change the value to 2_00 and make the remainder 0
-    if(decimals == 100) {
-        decimals = 0;
-        value += 1;
-    }
-    const formattedDecimals = decimals < 10 ? `0${decimals}` : `${decimals}`;
+export function formatMoney(value: number, currencyCode = "€", separators: {decimal?: string, thousand?: string} = {}) {
+    // First obtain formatted value with '.' as decimal separator and ',' as thousand separator (US notation).
+    let formattedValue = (Math.ceil(value) / 100).toLocaleString("en-US", {minimumFractionDigits: 2, maximumFractionDigits: 2});
```

Comment:
I kept the `Math.ceil` here, but I was wondering if this makes sense? You can't really pay a 0.1 c€ right? So I wondered if this shouldn't be replaced by `Math.round` or even `Math.floor`.